REPL.cpp

```diff
@@ -24,6 +36,14 @@ REPL::REPL() : m_diagnostic_engine(m_src_mgr),
     SetupIROpts();
     SetupImporters();
     swift::registerTypeCheckerRequestFunctions(m_ast_ctx->evaluator);
+
+    auto jit = JIT::Create();
+    if(!jit)
+    {
+        std::cerr << "Failed to initialize JIT\n";
+        return;
+    }
+    m_jit = std::move(*jit);
```

Comment:
2 things:
1. I've noticed that the JIT isn't used at all. Why does REPL have one? Are you making one for a side effect or something like that?
2. If the JIT fails to initialize, how can the caller know if the REPL it gets back is alright or ready to use? I recommend making the constructor private or protected and having a static function that makes a REPL for you and can verify that it was created with no problems before it hands it back. It could return an llvm::Expected or something like that.

Reply:
Yeah, you're right. JIT will be used later, but not quite yet. I'll remove it for this PR. I'll change constructor to return llvm::Expected.

Reply:
Constructors can't return anything. You'll want a static method in REPL that constructs a REPL and returns an llvm::Expected.

Reply:
Yeah that's what I meant - JIT has one of those right now.